src/statmodels.jl

```diff
@@ -200,6 +200,13 @@ const adjr² = adjr2
 abstract type RegressionModel <: StatisticalModel end
 
 """
+    coefnames(obj::RegressionModel)
+
+Return the names of the coefficients as a `Vector{Symbol}`.
```

Comment:
I wouldn't specify that it's a `Vector{Symbol}`. As noted on the issue, it could also be a `NamedTuple` for types with fixed parameters.

Reply:
I was thinking having the same expected behavior since it has the same function name as DataFrames / StatsModels, but I guess for some packages the flexibility of specifying the return type overweights this.